wiki/Beatmapping/Hitsound/vi.md

```diff
@@ -9,6 +9,8 @@ tags:
   - hitsound additions
   - hit sounds
   - hitsounds
+  - âm thanh
```

Comment:
delete this because hitsound cant really be translated to vietnamese

Reply:
ok thanks

Reply:
um actually i do have some other problems to fix so plz wait me